python/nnvm/testing/detection.py

```diff
@@ -0,0 +1,246 @@
+# pylint: disable=invalid-name, unused-variable, unused-argument, no-init
```

Comment:
the filename: detection.py could be more explicit, e.g. darknet_detection or darknet_yolo2 something.